go/vt/vtgate/buffer/flags.go

```diff
@@ -226,3 +226,8 @@ func (cfg *Config) bufferingMode(keyspace, shard string) bufferMode {
 
 	return bufferModeDisabled
 }
+
+// SetBufferingMode should only be used from testing code to change the flag default value
+func SetBufferingMode(enabled bool) {
+	*bufferEnabled = enabled
```

Comment:
I don't like this being an exported func in a common place like this. Maybe move it to a test_util.go and name it explicitly so that no one is tempted to use it outside of tests?

Reply:
Okay, done!